mindjunkies/forums/tests/test_views.py

```
@@ -558,34 +721,61 @@ def setUp(self):
-        """Test POST with valid reply form data"""
+        """Test POST request with valid reply form data"""
         request = self.factory.post("/", {"body": "New reply body"})
         request.user = self.user
-
-        # Add messages support to request
         setattr(request, "session", "session")
         messages = FallbackStorage(request)
         setattr(request, "_messages", messages)
-
-        
+        response = ReplyFormView.as_view()(
```

Ruff (0.8.2)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)